test/integration/query-tests/regressions/mapbox-gl-js#4494/style.json

```diff
@@ -2,6 +2,9 @@
   "version": 8,
   "metadata": {
     "test": {
+      "ignored": {
```

Comment:
See https://github.com/mapbox/mapbox-gl-js/pull/4604 for an alternative solution for this one.